ops/scripts/update-op-geth.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+PROJECTS="op-service op-node op-proposer op-batcher op-bindings op-chain-ops op-e2e"
+GETH_VERSION=v1.10.23
+
+export GOPRIVATE="github.com/ethereum-optimism"
+
+for value in $PROJECTS
+do
+	echo "Updating $value"
+	cd "./$value"
+	go mod edit -replace "github.com/ethereum/go-ethereum@$GETH_VERSION=github.com/ethereum-optimism/op-geth@optimism-history"
+	go mod tidy
+	cd ../
+done
```

Comment:
Let's add a `go work edit -replace ...........`

Reply:
ah good call, always forget that.